README.rst

```diff
@@ -1,4 +1,21 @@
 MagentoSyncModule
 =================
 
-Magento module for data sync and user login save to cookie
+Magento module for data sync and user login save to cookie.
+
+Module saves date cookies ``ongr_cart`` and ``ongr_user`` in json format. ``ongr_cart`` contains list of products in
+cart where array key is product id and value amount of products. Cookie is updated every time cart is saved or user logs
+in. ``ongr_user`` contains information about currently logged in user in object with ``id`` and ``data`` where id is
```

Comment:
``ongr_user`` contains information about user currently logged in. It is a object with properties ``id`` and ``data``, where id is